third_party/move/move-compiler-v2/transactional-tests/tests/control_flow/local_assigned_many_times.exp

```diff
@@ -1,3 +1,12 @@
-processed 1 task
-
-==> Compiler v2 delivered same results!
+comparison between v1 and v2 failed:
+= processed 1 task
+= 
++ task 0 'run'. lines 1-30:
++ warning: Unused assignment to `y_ref`. Consider removing or prefixing with an underscore: `_y_ref`
```

Comment:
Looks like v1 was imprecise here.

Reply:
Debatable.  The value was assigned to `_` at  `local_assigned_many_times.move:18` and `:22`, so it was used, in some sense.  But being assigned to `_` it was discarded and had no side-effects, so we removed it.  Unfortunately, the AST pass removed those uses (as not being useful) but not the assignment.